runtime/src/parachains.rs

```diff
@@ -173,7 +176,7 @@ impl<AccountId, T: Currency<AccountId>> ParachainCurrency<AccountId> for T where
 	}
 }
 
-pub trait Trait: session::Trait {
+pub trait Trait: session::Trait + staking::Trait {
```

Comment:
i'd rather have an associated `trait StatementRewards`. Can you place it in the new `attestations.rs`? Then it's in the right place to reward after-the-fact attestations when that's implemented.